avm/ptn/lz/sub-vending/tests/e2e/vwan-spoke/main.test.bicep

```diff
@@ -0,0 +1,77 @@
+metadata name = 'Vwan topology.'
+metadata description = 'This instance deploys a subscription with a vwan network topology.'
+
+targetScope = 'managementGroup'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = 'uksouth'
```

Comment:
Please note, this value (in this and other files) will be overwritten by the pipeline